### DISCUSSION TITLE
Include TouchSensor by default

### DISCUSSION THREAD

**Discussion by srmagura** [clauderic/dnd-kit#434] [Q&A] [answered]
The [DndContext documentation](https://docs.dndkit.com/api-documentation/context-provider#sensors) states that only the Pointer and Keyboard sensors are enabled by default. Why isn't `TouchSensor` included by default? Does it cause some negative side effects when not on a touch device?

Ideally, `dnd-kit` would support mobile devices out of the box without any additional configuration.

**Comment by clauderic** [comment, marked as answer]
Touch devices are supported out of the box via the Pointer sensor.

[Pointer events](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events) are designed to create a single DOM event model to handle pointing input devices such as a mouse, pen/stylus or touch.

The `TouchSensor` is available if you need to support browsers that don't have support for Pointer events or if you'd like different activation constraints for touch devices than mouse devices.

**Reply by srmagura** [reply]
Thanks for the clarification. Please see issue https://github.com/clauderic/dnd-kit/issues/435.
